clang_build/target.py

```diff
@@ -41,7 +37,7 @@ def name(self):
 
     @property
     def identifier(self):
-        """Return a unique identifier of this target.
```

Comment:
Not sure why this was changed. According to D401 (http://www.pydocstyle.org/en/2.1.1/error_codes.html), the first line should start with an imperative. If you have a good reason to deviate from this, I am fine with the change, otherwise I would stick with what people know and expect.